assets/css/contact.css

```diff
@@ -3,7 +3,10 @@
 
 /** bymeacoffee**/
 body{
-    font-family: 'Montserrat', sans-serif;
+    font-family: 'Lato', sans-serif;
```

Comment:
did you change this Montserrat or Poppins

Reply:
Hey @anushbhatia , I have implemented two fonts: Arvo (for the headings) and Lato (for the body) across all the pages as both of them together suit the theme of Doc2Pen. More over they contrast well with each other. 

Now, to maintain uniformity, I have changed fonts according to their role as Headings/Titles and Text. But I have accidentally changed the Buy me a coffee, as the original looks good. I'll unstage and correct it!